packages/react-router/src/link.tsx

```diff
@@ -1,3 +1,5 @@
+'use client'
```

Comment:
Does this currently work with RSCs? or is it user-land `"use client"` components that are wonky?